utils/recipients.py

```diff
@@ -220,8 +227,25 @@ def validate_email_address(email_address):
         raise InvalidEmailError('Not a valid email address')
 
 
+def format_email_address(email_address):
+    return email_address.strip().lower()
+
+
 def validate_recipient(recipient, template_type):
     return {
         'email': validate_email_address,
         'sms': validate_phone_number
     }[template_type](recipient)
+
+
+def format_recipient(recipient):
+    try:
+        return validate_and_format_phone_number(recipient)
+    except InvalidPhoneError:
```

Comment:
Almost. The `format_email_address` method doesn’t do any validation so will never raise an `InvalidEmailError`. So there’s no need to catch it.